src/modules/70-pipeline/pages/inputSet-list/InputSetListView.tsx

```diff
@@ -136,6 +137,7 @@ const RenderColumnMenu: Renderer<CellProps<InputSetLocal>> = ({ row, column }) =
   const data = row.original
   const [menuOpen, setMenuOpen] = React.useState(false)
   const { getString } = useStrings()
+  const isPipelineInvalid = (column as any)?.isPipelineInvalid
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9275140)
```suggestion
  const isPipelineInvalid = (column as unknown)?.isPipelineInvalid
```